build/submodules.props

```diff
@@ -36,44 +36,47 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Repository Include="AADIntegration" />
-    <Repository Include="Antiforgery" />
-    <Repository Include="AzureIntegration" />
-    <Repository Include="BasicMiddleware" />
-    <Repository Include="BrowserLink" />
-    <Repository Include="CORS" />
-    <Repository Include="DataProtection" RootPath="$(RepositoryRoot)src\DataProtection\" />
-    <Repository Include="Diagnostics" />
-    <Repository Include="DotNetTools" />
-    <Repository Include="EntityFrameworkCore" />
-    <Repository Include="Hosting" />
-    <Repository Include="HtmlAbstractions" />
-    <Repository Include="HttpAbstractions" />
-    <Repository Include="HttpClientFactory" />
-    <Repository Include="HttpSysServer" />
-    <Repository Include="Identity" />
-    <Repository Include="IISIntegration" RootPath="$(RepositoryRoot)src\IISIntegration\" />
-    <Repository Include="JavaScriptServices" />
-    <Repository Include="JsonPatch" />
-    <Repository Include="KestrelHttpServer" />
-    <Repository Include="Localization" />
-    <Repository Include="MetaPackages" PatchPolicy="CascadeVersions" />
-    <Repository Include="Mvc" />
-    <Repository Include="MvcPrecompilation" />
-    <Repository Include="Razor" />
-    <Repository Include="ResponseCaching" />
-    <Repository Include="Routing" />
     <Repository Include="Scaffolding" PatchPolicy="AlwaysUpdate" />
-    <Repository Include="Security" />
-    <Repository Include="Session" />
-    <Repository Include="SignalR" />
-    <Repository Include="StaticFiles" />
+    <Repository Include="IISIntegration" RootPath="$(RepositoryRoot)src\IISIntegration\" />
```

Comment:
Shouldn't this be in the ItemGroup below?

Reply:
I was under the impression there were changes in IISIntegration in 2.2.1. If that's not true, then yes, this should be reverted.

Reply:
Got it.